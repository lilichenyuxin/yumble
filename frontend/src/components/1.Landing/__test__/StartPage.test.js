import React from 'react';
import renderer from 'react-test-renderer';
import StartPage from '../StartPage';
import {BrowserRouter} from 'react-router-dom';

test('Start Page snapshot testing', () => {
  const component = renderer.create(
      <BrowserRouter>
        <StartPage />
      </BrowserRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
