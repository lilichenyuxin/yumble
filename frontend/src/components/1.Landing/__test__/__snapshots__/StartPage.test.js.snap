// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Start Page snapshot testing 1`] = `
Array [
  <div
    className="MakeCentre"
  >
    <h1
      className="StartTitle"
    >
      yumble
    </h1>
    <a
      href="/Preferences"
      onClick={[Function]}
    >
      <button
        className="BigBtn"
        id="CreateGroup_btn"
      >
        Create group
      </button>
    </a>
    <a
      href="/JoinGroup"
      onClick={[Function]}
    >
      <button
        className="BigBtn"
        id="JoinGroup_btn"
      >
        Join group
      </button>
    </a>
  </div>,
  <button
    className="SmallBtn"
    id="HelpButton"
    onClick={[Function]}
  >
    help?
  </button>,
  "",
]
`;
